estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,166 @@
+                </header>
+                <sheet>
+                    <group>
+                       <field name="name" string="Name"/>
+                    </group>
+                    <group>
+                        <field name="tag_ids" widget="many2many_tags" nolabel="1"/>
+                    </group>
+                    <group>
+                        <group>
+                            <field name="property_type_id" string='Property Type'/>
+                        </group>
+                        <group>
+                            <field name="expected_price"/>
+                        </group>
+                    </group>
+                    <group>
+                        <group>
+                            <field name="postcode"/>
+                        </group>
+                        <group>
+                            <field name="best_price"/>
+                        </group>
+                    </group>
+                    <group>
+                        <group>
+                            <field name="date_availability"/>
+                        </group>
+                        <group>
+                            <field name="selling_price"/>
+                        </group>
+                    </group>
+                   <notebook>
+                        <page string="Description">
+                            <group>
+                                <field name="description"/>
+                                <field name="bedrooms"/>
+                                <field name="living_area" string="Living Area (sqm)"/>
+                                <field name="facades"/>
+                                <field name="garage"/>
+                                <field name="garden"/>
+                                <field name="garden_area" string="Garden Area (sqm)"/>
+                                <field name="garden_orientation"/>
+                                <field name="total_area"/>
+                            </group>
+                        </page>
+                        <page string="Offers">
+                            <group>
+                                <field name="offer_ids" nolabel="1"/>
+                            </group>
+                        </page>
+                        <page string="Other Info">
+                            <group>
+                                <field name="user_id"/>
+                                <field name="partner_id"/>
+                            </group>
+                        </page>
+                        
+                   </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="estate_property_offer_view_form" model="ir.ui.view">
+        <field name="name">estate.property.offer.form</field>
+        <field name="model">estate.property.offer</field>
```

Comment:
Create a new file `estate_property_offer.xml`.And there you can add views for `estate.property.offer`  to easy code readability.